PEERS_GUIDELINES.md

```diff
@@ -118,57 +85,44 @@ Please specify which of the following two locales you wish to have unfrozen:
 - German (de)
 - Polish (pl)
 
-NB These are the only frozen locales we will consider - see
-[The future of MDN localization in this Hacks post](https://hacks.mozilla.org/2020/10/mdn-web-docs-evolves-lowdown-on-the-upcoming-new-platform/)
+NB These are the only frozen locales we will consider - see [The future of MDN localization in this Hacks post](https://hacks.mozilla.org/2020/10/mdn-web-docs-evolves-lowdown-on-the-upcoming-new-platform/)
 
 ### Your team members
 
 You need to put together a community maintenance team. This requires:
 
-- A team lead who will be the communication point between that team and the MDN
-  core team, and have overall responsibility for the team.
+- A team lead who will be the communication point between that team and the MDN core team, and have overall responsibility for the team.
 - At least two other members, so that members can review other members work.
-- A place to discuss this team's localization work. This can be a Telegram
-  group, Matrix chat room, or whatever the team thinks is best.
+- A place to discuss this team's localization work. This can be a Telegram group, Matrix chat room, or whatever the team thinks is best.
 
-To find out more about our existing teams see [Review teams](#review-teams)
-above.
+To find out more about our existing teams see [Review teams](#review-teams) above.
 
 ### Agree to our terms
 
-Please indicate that you agree to our
-[Requirements for keeping locales up-to-date](#requirements-for-keeping-locales-up-to-date)
- above.
+Please indicate that you agree to our [Requirements for keeping locales up-to-date](#requirements-for-keeping-locales-up-to-date) above.
 
 ### The team agrees to our CoC and license
 
-Also indicate that your team agrees to adhere by our Code of Conduct and License
-(see below)
+Also indicate that your team agrees to adhere by our Code of  and License (see below)
```

Comment:
```suggestion
Also indicate that your team agrees to adhere by our Code of Conduct and License (see below)
```

Reply:
Thanks, sorry for the inattention here :/